target/ndeploy.framework.symfony2.xml

```diff
@@ -34,7 +34,15 @@
                     <istrue value="${application.framework.extra.migrate.ask}"/>
                     <then>
                         <echo msg="Checking doctrine migrations." />
-                        <exec dir="${application.releasesdir.latest}" command="${application.framework.extra.symfony2.command} doctrine:migrations:status --show-versions" outputProperty="migration-versions" checkreturn="true" />
+                        <if>
+                            <equals arg1="${application.framework.extra.migrate.version}" arg2="3" />
```

Comment:
please add the new parameter to the "ndeploy.framework.symfony2.config" target too (somewhere at the end of this file)